src/umpire/tpl/json/json.hpp

```diff
@@ -2410,7 +2410,7 @@ class input_buffer_adapter : public input_adapter_protocol
     input_buffer_adapter& operator=(input_buffer_adapter&) = delete;
     input_buffer_adapter(input_buffer_adapter&&) = delete;
     input_buffer_adapter& operator=(input_buffer_adapter&&) = delete;
-    ~input_buffer_adapter() override = default;
+    ~input_buffer_adapter() override {};
```

Comment:
What was the problem here?

Reply:
This was generating a "undefined reference to `vtable for nlohmann::detail::input_buffer_adapter'" error at link time for the replay test. Some Google'ing suggested that some compilers don't like overriding a pure-virtual destructor with a defaulted function. Strange issue, but this change seemed to resolve it.

Reply:
Very odd! The `;` is not needed.

Reply:
Did this get removed?